src/harness/reference_models/interference/aggregate_interference.py

```diff
@@ -207,12 +201,11 @@ def calculateAggregateInterferenceForGwpz(gwpz_record, cbsd_list):
 
   Args:
     gwpz_record: GWPZ protection entity
-    cbsd_list: list of CBSD objects containing registrations and grants
+    cbsd_list: list of CbsdGrantInfo objects containing registrations and grants
   Returns:
     Aggregate interference to GWPZ in the nested dictionary format.
       {latitude : {longitude: [aggr_interf1(mW), ..., aggr_interfK(mW)]}}
   """
-  grant_list = data.getAllGrantInfoFromCbsdDataDump(cbsd_list)
   gwpz_region = gwpz_record['zone']['features'][0]['properties']['clutter']
```

Comment:
'clutter' shall be changed to 'landCategory' as per discussion with Kate and Serge.

Reply:
Done.